Monika After Story/game/zz_calendar.rpy

```diff
@@ -548,11 +549,17 @@ M̼̤̱͇̤ ͈̰̬͈̭ͅw̩̜͇͈ͅa̲̩̭̩ͅs̙ ̣͔͓͚̰h̠̯̫̼͉e̗̗̮r
 
             # get this month's events
             if self.MIN_GLITCH_YEAR < self.selected_year < self.MAX_GLITCH_YEAR:
-
                 events = self.database[self.selected_month]
 
-            else:
+                #For leap year case, we put F29 into M01
+                if self.selected_month == 3 and self.selected_year % 4 != 0:
```

Comment:
mm unfortunate that this isn't true if the year is divisible by 100 unless its divisible by 400.